content/api/vision/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Vision
+---
+
+Your webcam is how your robot will see markers.
+
+## Searching for markers
+Assuming you have a webcam connected, you can use `r.camera.see()` to take a picture. The software will process the picture and returns a list of the markers it sees, sorted by distance, where the first element of the array is the closest marker to the camera.
+
+```python
+markers = r.camera.see()
+```
+
+## Marker properties
+The markers in the list have some useful properties:
+
+- `is_wall_marker()` - returns whether or not the marker is a [wall marker](marker-ids/#wall-markers).
+- `is_token_marker()` - returns whether or not the marker is a [token marker](marker-ids/#token-markers).
+- `id` - returns the [id](marker-ids) of the marker.
+- `pixel_centre` - returns the pixel coordinates of the centre of the marker.
```

Comment:
I think it'd be a good idea to define "pixel coordinates", as the meaning of the phrase by itself isn't very obvious.